db/db_options_test.cc

```diff
@@ -797,6 +798,210 @@ TEST_F(DBOptionsTest, InMemoryStatsHistoryPurging) {
   Close();
 }
 
+int countkeys(Iterator* iter) {
+  int count = 0;
+  for (iter->SeekToFirst(); iter->Valid(); iter->Next()) {
+    // if (count % 100 == 0) {
+    //   fprintf(stdout, "key = %s, value = %s\n",
+    //   iter->key().ToString().c_str(), iter->value().ToString().c_str());
+    // }
```

Comment:
It's good that you do some manual testing and make sure the outputs are correct. It's a good idea to at least put some of it as a part of the test itself. We need to make sure the data itself in the stat CF is correct, as part of the regression test.